website/docs/how-to/how-to-environment-import-export.mdx

```diff
@@ -3,9 +3,9 @@ title: Environment Import & Export
 ---
 import VideoContent from '@site/src/components/VideoContent.jsx'
 
-:::info Availability
+:::note Availability
 
-The environment import and export first appeared in Unleash 4.22.0.
+**Unleash version**: `4.22+`
```

Comment:
Did you leave "Unleash" in by mistake here? All other places you just say "version" unless there's other systems available too (i.e. the proxy)

```suggestion
**Version**: `4.22+`
```